flink-python/pyflink/table/tests/test_descriptor.py

```diff
@@ -963,7 +963,7 @@ def test_register_table_source_and_register_table_sink(self):
                           .field("a", DataTypes.INT())
                           .field("b", DataTypes.STRING())
                           .field("c", DataTypes.STRING()))\
-             .register_table_source("source")
+             .create_temporary_table("source")
```

Comment:
What about removing test_register_table_source_and_register_table_sink and renaming test_register_table_source_and_sink to test_register_temporary_table?

Reply:
Nice, will do.